contracts/SaveDAI.sol

```diff
@@ -65,6 +60,12 @@ contract SaveDAI is ERC20, ERC20Detailed, ERC20Pausable, Ownable {
         uniswapFactory = UniswapFactoryInterface(uniswapFactoryAddress);
         daiUniswapExchange = _getExchange(daiAddress);
         ocDaiExchange = _getExchange(ocDaiAddress);
+
+        require(
```

Comment:
@dougiebuckets are we not adding error messages bc if it reverts we will see the error message from the ERC20 contract?

Reply:
@kseniya292 Yeah, exactly.

Reply:
Right, and this reverse-reason would only be used if the approve returns false, not if the approve call reverts. A revert message here would only be used if approve succeeded as a call, but returned false.